Bottom: acfb32528de8f51be884ac2c5cfc593134cae8d1
Top:    5851def43be697a39cb8a99adbbd85a430f4e353
Author: Hal Rosenstock <hal@dev.mellanox.co.il>
Date:   2013-09-11 14:44:32 -0400

[librdmacm] rstream.c: Indicate when specified address family is unknown

Signed-off-by: Hal Rosenstock >hal@mellanox.com>
Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/examples/rstream.c b/examples/rstream.c
index 278437f..cf84716 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -586,6 +586,8 @@ int main(int argc, char **argv)
 				rai_hints.ai_flags = RAI_NUMERICHOST | RAI_FAMILY;
 				rai_hints.ai_family = AF_IB;
 				use_rgai = 1;
+			} else {
+				fprintf(stderr, "Warning: unknown address format\n");
 			}
 			break;
 		case 'B':
